packages/cli/src/cmds/compare/ChangeReporter.ts

```diff
@@ -1,15 +1,15 @@
 import { mkdir, readFile, rm, writeFile } from 'fs/promises';
 import { default as openapiDiff } from 'openapi-diff';
-import { dirname, isAbsolute, join, relative } from 'path';
+import { dirname, isAbsolute, join, relative, resolve } from 'path';
 import { ClassMap, Metadata } from '@appland/models';
 import { FormatType, format } from '@appland/sequence-diagram';
 import { queue } from 'async';
 import assert from 'assert';
 
 import { DiffDiagrams } from '../../sequenceDiagramDiff/DiffDiagrams';
 import { ArchiveMetadata } from '../archive/ArchiveMetadata';
-import { AppMapData } from './AppMapData';
-import { AppMapIndex } from './AppMapIndex';
+import { Paths } from './Paths';
+import { Digests } from './Digests';
```

Comment:
I've chosen better names here, that's all. The name change is isolated to one commit.